usecase/call_rpc.go

```diff
@@ -416,6 +429,44 @@ func (m *dependencyManager) CallRPC(ctx context.Context, w io.Writer, rpcName st
 	}
 }
 
+// Gets a request with the body containing the payload of its previous RPC
+// Only RPCs that are repeatable by definition will have their previous requests returned.
+func (m *dependencyManager) getPreviousRPCRequest(rpcName string, req interface{}) error {
+	id := rpcIdentifier(rpcName)
+	if m.state.rpcCallState[id].repeatable {
+		previousReqBytes := m.state.rpcCallState[id].reqPayload
+		if previousReqBytes == nil {
+			return errors.New(fmt.Sprintf("No previous request body exists for RPC: %s. Please issue a normal request.", rpcName))
```

Comment:
According to [Go CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#error-strings), we should be:

```suggestion
			return errors.Wrapf("no previous request body exists for RPC: %s, please issue a normal request", rpcName)
```

Reply:
:D New Gopher here. Thank you for guiding me on the comment standard. Helpful. 
Although I believe there exists no `Wrapf()` that takes in no error. You probably meant `Errorf`)

Reply:
> Although I believe there exists no Wrapf() that takes in no error. You probably meant Errorf)

Oops, it's right!